internal/compiler/langtype.rs

```diff
@@ -266,6 +270,8 @@ impl Type {
             | (Type::Int32, Type::Model)
             | (Type::PhysicalLength, Type::LogicalLength)
             | (Type::LogicalLength, Type::PhysicalLength)
+            | (Type::Rem, Type::LogicalLength)
+            | (Type::Rem, Type::PhysicalLength)
```

Comment:
```suggestion
            | (Type::LogicalLength, Type::Rem)
            | (Type::PhysicalLength, Type::Rem)
```

Reply:
Looks like the conversion to rem is still missing

Reply:
Yes indeed. I'll add that (instead of removing rem to px/phx), as well as test coverage.